src/components/application_manager/rpc_plugins/rc_rpc_plugin/CMakeLists.txt

```diff
@@ -45,10 +45,12 @@ set (HMI_COMMANDS_SOURCE_DIR
 collect_sources(MOBILE_COMMANDS "${MOBILE_COMMANDS_SOURCE_DIR}")
 collect_sources(HMI_COMMANDS "${HMI_COMMANDS_SOURCE_DIR}")
 collect_sources(PLUGIN_SOURCES "${CMAKE_CURRENT_SOURCE_DIR}/src")
+collect_sources(COMMANDS_SOURCES "${CMAKE_CURRENT_SOURCE_DIR}/src/commands")
```

Comment:
@LitvinenkoIra is there hmi and mobile splitting?

Reply:
@LuxoftAKutsan yes